### PR TITLE
Pup : Command line html parser

### PR DESCRIPTION
**Pull request by kcubeterm**
#4995

### CODE REVIEW

File: packages/pup/build.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        export GOPATH="${TERMUX_PKG_BUILDDIR}"
+        mkdir -p "${GOPATH}/src/github.com/ericchiang/"
+        cp -a "${TERMUX_PKG_SRCDIR}" "${GOPATH}/src/github.com/ericchiang/pup"
+        cd "${GOPATH}/src/github.com/ericchiang/pup"
+        go get -d -v
+        go build
+
+}
+
+        termux_step_make_install() {
```
Comment (xeffyr) [inline packages/pup/build.sh:23]: Fix formatting.

**Comment by kcubeterm** [comment]
@xeffyr , why its not passed, i fixed the format only

**Comment by xeffyr** [comment]
I merged it before PR checks passed.
Ignore it as package already in repository, `pkg install pup`.

**Comment by kcubeterm** [comment]
Oo

**Pull request merged**